### ISSUE TITLE
A11y_TrydotNetWebsite_AI4W: Ensures each HTML document contains a non-empty <title> element (html)

### ISSUE THREAD

**Issue by Sobi261** [dotnet/try#1007]
"[[Check out Accessibility Insights!](https://accessibilityinsights.io/)] - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHub Tags:**
#A11yAuto; #A11ySev2; #A11yMAS; #A11yTCS; #Benchmark; #DesktopWeb; #Win11; #ChromiumEdge; #P2-TrydotNetWebsite-Aug21; #AI4W; #WCAG2.4.2; 

### Environment Details:
Application Name: TrydotNetWebsite

### OS and Browser Details:
Chromium Edge Version 99.0.1150.52 (Official build) (64-bit)
Windows 11

### Repro Steps:

1. Hit the URL: https://trydotnet.microsoft.com
2. Open AI4W tool. Run Automated Checks

### Issue: 
Ensures each HTML document contains a non-empty <title> element (document-title - https://accessibilityinsights.io/info-examples/web/document-title)

### Target application: 
https://try.dot.net - https://try.dot.net/

### Element path:
 html

### Snippet: 
<html lang="en">

### How to fix: 
Fix any of the following:
  Document does not have a non-empty <title> element

### Environment: 
Microsoft Edge version 101.0.1210.47

====

This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Attachment:
![A11y_TrydotNetWebsite_AI4W](https://user-images.githubusercontent.com/102137405/168822884-4cbb4fea-debe-40cc-af1b-af045b790db6.png)

**Comment by fsteffi** [comment]
does not reproduce
![image](https://user-images.githubusercontent.com/24580570/210364332-932f9ce5-2d2b-4a1d-a99b-d31b1fe324df.png)
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha

**Comment by AbhitejJohn** [comment]
Thanks @fsteffi . It seems like the issue you have in your screenshot is tracked by https://github.com/dotnet/try/issues/1006. Closing this out.
